src/components/Courses/components/CourseCard/CourseCard.tsx

```diff
@@ -40,7 +40,7 @@ const CourseCard = ({ courseId }) => {
 	};
 	if (course) {
 		return (
-			<div className='course-card'>
+			<div className='course-card' data-testid='course-card'>
```

Comment:
data-testid attribute is the less preferrable way to find elements in tests (see priority section here: https://testing-library.com/docs/queries/about/). Your test should resemble the way how real user interacts with the page, i.e. query only by visible text and/or role.